f2/src/forms/BusinessInfoForm.js

```diff
@@ -1,60 +1,86 @@
-import { FormHelperText } from '../components/FormHelperText'
-import { FormLabel } from '../components/FormLabel'
+import { Stack } from '@chakra-ui/core'
 import { useStateValue } from '../utils/state'
+import { Input } from '../components/input'
+import { Field } from '../components/Field'
+import { FormArrayControl } from '../components/FormArrayControl'
+import { RadioAdapter } from '../components/radio'
 
-export const BusinessInfoForm = props => {
-  const localOnSubmit = data => {
-    props.onSubmit(data)
-  }
-
+export const BusinessInfoForm = ({ onSubmit }) => {
   const [data] = useStateValue()
   let { businessInfo } = data.formData
   businessInfo = {
     business: '',
     ...businessInfo,
   }
   return (
-    <Form
-      initialValues={businessInfo}
-      onSubmit={data => localOnSubmit(data)}
-      render={({ handleSubmit }) => (
-        <Stack as="form" onSubmit={handleSubmit} shouldWrapChildren spacing={6}>
-          <Field name="business">
-            {props => (
-              <FormControl>
-                <FormLabel htmlFor="business">
-                  <Trans id="businessPage.business" />
-                </FormLabel>
-                <FormHelperText>
-                  <Trans id="businessPage.businessExample">
-                    <VisuallyHidden as="span" />
-                  </Trans>
-                </FormHelperText>
-                <TextArea
-                  id="business"
-                  name={props.input.name}
-                  value={props.input.value}
-                  onChange={props.input.onChange}
-                />
-              </FormControl>
-            )}
-          </Field>
+    <React.Fragment>
+      <Form
+        initialValues={businessInfo}
+        onSubmit={onSubmit}
+        render={({ handleSubmit }) => (
+          <Stack
+            as="form"
+            onSubmit={handleSubmit}
+            shouldWrapChildren
+            spacing={6}
+          >
+            <Field
+              name="nameOfBusiness"
+              label={<Trans id="businessPage.nameOfBusiness" />}
+              component={Input}
+            />
+
+            <Field
+              name="industry"
+              label={<Trans id="businessPage.industry" />}
+              helperText={<Trans id="businessPage.industryExample" />}
+              component={Input}
+            />
+
+            <Field
+              name="role"
+              label={<Trans id="businessPage.role" />}
+              helperText={<Trans id="businessPage.roleExample" />}
+              component={Input}
+            />
+
+            <FormArrayControl
+              name="numberOfEmployee"
+              label={<Trans id="businessPage.numberOfEmployee" />}
+              helperText={<Trans id="businessPage.numberOfEmployeeExample" />}
+            >
+              <RadioAdapter name="numberOfEmployee" value="1 to 99 employees">
+                1 to 99 employees
```

Comment:
We should translate this text, and the other radios! With a <Trans> tag

Reply:
fixed